src/App/Order/Calculator/General/LabelDescriptionCalculator.php

```diff
@@ -31,34 +32,38 @@ private function calculateDescription(): string
             return implode(', ', Utils::filterNull(Arr::pluck($this->order->lines->all(), $key)));
         };
 
-        return preg_replace_callback_array([
-            '/\[ORDER_ID\]/' => function () {
-                return $this->order->externalIdentifier;
-            },
+        return substr(
```

Comment:
`Str::limit`, en dan liever het resultaat van `preg_replace_callback_array` even in een variabele zetten om het extra indent level te vermijden en het leesbaarder te maken